### ISSUE TITLE
[Question] Step 3 loss curve

### ISSUE THREAD

**Issue by fanminshi** [PKU-YuanGroup/MoE-LLaVA#89]
### Question

I performed the step 3 moe-finetunning on phi-2 model, the loss doesn't seemed to drop that much. I wonder if that's normal. Thanks!

<img width="1452" alt="Screenshot 2024-08-16 at 8 29 19 AM" src="https://github.com/user-attachments/assets/9816300a-81c2-48e1-999b-7b9af02ea19a">

**Comment by EmberNoWeither** [comment]
Hello! When i use the script 'finetune_moe.sh' to train phi-2, i got the similar curves! But I found a problem that my training process will be OOM! It happened randomly and the gpu memory-usage is not constant. The gpu memory-usage will vary widely leading to OOM... Did you find this problem?
